handlers/main.yml

```diff
@@ -40,12 +48,20 @@
     state: "reloaded"
   when: "ansible_distribution_version | version_compare('16.04', '==')"
 
+- name: "Reload rh-php70-php-fpm"
+  service:
+    name: "rh-php70-php-fpm"
+    state: "reloaded"
+  when: ansible_os_family == "RedHat"
+
 - name: "Clear sf_cache"
   shell: "php symfony cc"
   args:
     chdir: "{{ atom_path }}"
   become_user: "{{ atom_user }}"
-  environment: "{{ atom_pool_php_envs }}"
+  environment:
+    - "{{ atom_pool_php_envs }}"
+    - PATH: "{{ ansible_env.PATH }}:/opt/rh/rh-php70/root/bin/"
```

Comment:
Similar to tasks/flush.yml, does this PATH change (/opt/rh/rh-php70/root/bin) even occur if not CentOS/RedHat? Possibly harmless if so, or should there be a conditional?